scheduler/src/cook/components.clj

```diff
@@ -325,7 +328,12 @@
 (def config-settings
   "Parses the settings out of a config file"
   (graph/eager-compile
-    {:server-port (fnk [[:config port]]
+    {:agent-query-cache (fnk [[:config {agent-query-cache nil}]]
+                          (merge
+                            {:max-size 1000
```

Comment:
Let's make the default larger, maybe 5000 so that most users won't need to set it.